### DISCUSSION TITLE
Is it possible to use a column from /SitePages in Filter webpart?

### DISCUSSION THREAD

**Discussion by TomLIKG** [microsoft-search/pnp-modern-search#3893] [Q&A] [answered]
Hey!

I've been trying to set up filtering for my SitePages for a few days now. I have a column with a selection list and want to use this as a filter for the website pages.

But I can't seem to assign this column to a RefinableString. 
Is there an alternative option here or is this not possible?

Thanks in advance!

I want to filter the column "Kategorie" in the filter webpart
![Screenshot 2024-08-09 080147](https://github.com/user-attachments/assets/4f9008e6-82e9-43db-a976-4ed71156f970)

**Comment by kasperbolarsen** [comment]
Can you specify the properties for that selection list?
Is it a Taxonomy field, a Choice field or a lookup in another list?

**Reply by TomLIKG** [reply]
Thank you for your message!
Its just a manually added selection list 
![image](https://github.com/user-attachments/assets/f85ed741-c05a-48f8-bfdc-42695e4ff1b0)

**Reply by TomLIKG** [reply]
Is there an existing column that I could use for this or do I have to create this column differently?

**Reply by kasperbolarsen** [reply, marked as answer]
No, a Choice field is OK, but I will always recommend that you create the field as a Site Column, not a List Column, as this will ensure that the crawled and managed property will be generated

**Reply by TomLIKG** [reply]
Thank you for your help @kasperbolarsen
